src/mon/Monitor.cc

```diff
@@ -5379,9 +5430,26 @@ void Monitor::tick()
     paxos->trigger_propose();
   }
 
+  check_ops_in_flight();
+
   new_tick();
 }
 
+void Monitor::check_ops_in_flight()
+{
+  string summary;
+  vector<string> warnings;
+  int slow = 0;
+  if (op_tracker.check_ops_in_flight(&summary, warnings, &slow)) {
+    clog->warn() << summary;
```

Comment:
i suggest avoid sending the slow request to cluster log, instead might want to use the new health reporting machinery to do this job:

1. collect the health info and send it to mgr in MMgrReport periodically
1. let mgr to process it 
1. so "ceph -s" will be able to alert the user.

Reply:
what's the disadvantage of sending the slow request to cluster log?

Reply:
If we send it to the manager, it can be used to swap health states. Going to the central log merely sends out an unparsed text statement.